unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/BackpackEditorHUDController.cs

```diff
@@ -418,7 +422,12 @@ private void UnEquipWearable(string wearableId, UnequipWearableSource source, bo
             if (source != UnequipWearableSource.None)
                 backpackAnalyticsController.SendUnequippedWearableAnalytic(wearable.data.category, wearable.rarity, source);
 
-            avatarSlotsHUDController.UnEquip(wearable.data.category);
+            //Reset override for affected categories of unequip
+            if(wearable.GetHidesList(ownUserProfile.avatar.bodyShape) != null)
```

Comment:
you should do a method for this block of code since its repeated in `EquipWearableFromGrid`